appcues/src/main/java/com/appcues/debugger/DebuggerRecentEventsManager.kt

```diff
@@ -98,6 +131,77 @@ internal class DebuggerRecentEventsManager(
         lastEventId++
     }
 
+    private fun onActivityGroup(activity: AnalyticsActivity) {
+        events.addFirst(
+            DebuggerEventItem(
+                id = lastEventId,
+                type = EventType.GROUP_UPDATE,
+                timestamp = Date().time,
+                name = activity.groupId ?: contextResources.getString(R.string.appcues_debugger_event_type_group_update_title),
+                propertySections = listOf(
+                    DebuggerEventItemPropertySection(
+                        title = contextResources.getString(R.string.appcues_debugger_event_details_properties_title),
+                        properties = activity.groupProperties?.toSortedList(),
+                    )
+                )
+            )
+        )
+        lastEventId++
+    }
+
+    private fun onActivityEvent(activity: AnalyticsActivity) {
```

Comment:
comparing this to the "old" method that gets an ActivityRequest as input, this one is straight forward since we know that every activity = an EventItem, whereas before we had to iterate the events list (that always contained ONE element at this moment)